habitat/sims/rearrangement/actions.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
```

Comment:
Rearrangement is not a sim, you may be extending functionality of Habitat-Sim, but it is not a Sim.